autocomplete/log.go

```diff
@@ -0,0 +1,28 @@
+package autocomplete
+
+import (
+	"fmt"
+	"os"
+)
+
+var logPath string
+
+func SetupLog(path string) {
```

Comment:
Should we add Godoc comments to exported functions?

Reply:
I favor commenting exported names. This used to be a pretty standard Go practice. 

> Every exported (capitalized) name should have a doc comment.

https://tip.golang.org/doc/comment

This code is simple, but comments aren't always for describing precisely what code does. The purpose of this code wasn't immediately obvious to me.

Reply:
> this code wasn't immediately obvious to me

that's a fair point; I'll add a comment mentioning it's optional and only used to trigger debug log output.

Reply:
Thanks!